### ISSUE TITLE
fade.lua SetAlpha errors

### ISSUE THREAD

**Issue by Fringetastic** [funkydude/BasicChatMods#23]
Bugsack had 10 errors that appear to originate from this addon, V8.2.11. They all were like this one:

```
1x BasicChatMods\fade.lua:24: Usage: ChatFrame2Background:SetAlpha(alpha)
[C]: in function `SetAlpha'
BasicChatMods\fade.lua:24: in function <BasicChatMods\fade.lua:17>
[C]: in function `SetAlpha'
FrameXML\FloatingChatFrame.lua:922: in function `FCF_SetWindowAlpha'
FrameXML\FloatingChatFrame.lua:193: in function `FloatingChatFrame_Update'
FrameXML\FloatingChatFrame.lua:112: in function `FloatingChatFrame_OnEvent'
[string "*:OnEvent"]:2: in function <[string "*:OnEvent"]:1>
...aceBlizzard_CombatLog\Blizzard_CombatLog.lua:3295: in function <...aceBlizzard_CombatLog\Blizzard_CombatLog.lua:3291>

Locals:
(*temporary) = ChatFrame2Background {
 0 = <userdata>
 SetAlpha = <function> defined =[C]:-1
}
(*temporary) = nil
```

The second line and (*temporary) line are the ones that changed in the bug captures. Other values:
ChatFrame2TopLeftTexture
ChatFrame2BottomLeftTexture
ChatFrame2TopRightTexture
ChatFrame2BottomRightTexture
ChatFrame2LeftTexture
ChatFrame2RightTexture
ChatFrame2BottomTexture
ChatFrame2TopTexture
ChatFrame2ButtonFrameBackground

**Comment by funkydude** [comment]
I cannot reproduce this, even with the combat log as a separate chat window. I'll need more info.

You should also try disabling all other addons to see if it still happens.

**Comment by Fringetastic** [comment]
It happens very inconsistently. I got it today on my AH toon, who doesn't have a separate window for a combat log or a combat log addon enabled. I got it yesterday on one login for one character, but I logged on to different characters multiple times without it happening twice. Logging on to the same character today, it didn't happen again. Now I'm wondering if it is occurring with the first login of the day. I will try to find a pattern.
